BASE/src/Microsoft.ApplicationInsights/DataContracts/AvailabilityTelemetry.cs

```diff
@@ -15,6 +15,7 @@
     /// </summary>
     public sealed class AvailabilityTelemetry : ITelemetry, ISupportProperties, ISupportMetrics, IAiSerializableTelemetry
     {
+        internal const string EtwEnvelopeName = "Availability";
```

Comment:
it may be the best to move these strings to the ETW channel itself.

Reply:
I see. My initial idea was to reduce the confusion for anyone looking into the code and assuming that new name is for everything. Const inside the telemetry type immediately lets the person know that there is more to envelope name than meets the eye.

It's used in two places: RichPayloadEventSource and RichPayloadEventSource.TelemetryHandler, so new home will still be something shared between these two, e.g. a separate constants file in the RPES namespace.

If you think we still need to proceed with moving out of telemetry classes, it's 5-min change, just let me know in response.

Reply:
@SergeyKanzhelev , did you have a change to take a look at my response?